src/devices/busch_jaeger_6737.ts

```diff
@@ -0,0 +1,51 @@
+import type { Definition } from '../lib/types';
```

Comment:
Add to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/busch_jaeger.ts instead